src/main/scala/it/pagopa/interop/catalogmanagement/service/CatalogFileManager.scala

```diff
@@ -1,23 +1,11 @@
 package it.pagopa.interop.catalogmanagement.service
 
 import akka.http.scaladsl.server.directives.FileInfo
-import cats.syntax.all._
-import com.typesafe.scalalogging.{Logger, LoggerTakingImplicit}
-import it.pagopa.interop.catalogmanagement.error.CatalogManagementErrors.{
-  DocumentAlreadyUploaded,
-  InvalidInterfaceFileDetected
-}
-import it.pagopa.interop.catalogmanagement.model.{CatalogDocument, CatalogItem, Rest, Soap}
+import it.pagopa.interop.catalogmanagement.model.CatalogDocument
 import it.pagopa.interop.commons.files.service.FileManager
-import it.pagopa.interop.commons.logging.{CanLogContextFields, ContextFieldsToLog}
-import it.pagopa.interop.commons.utils.Digester
-import org.apache.tika.Tika
-
 import java.io.File
-import java.nio.file.Files
 import java.util.UUID
 import scala.concurrent.{ExecutionContext, Future}
-import scala.util.Try
 
 trait CatalogFileManager {
```

Comment:
The `store` function is now useless, but I will remove it in a new story together with the CatalogFileManager deletion.